pkgs/applications/radio/rtl-sdr/default.nix

```diff
@@ -19,8 +19,7 @@ stdenv.mkDerivation rec {
   #   rules manually.
   # * Propagate libusb-1.0 dependency in pkg-config file.
   postInstall = stdenv.lib.optionalString stdenv.isLinux ''
-    mkdir -p "$out/etc/udev/rules.d/"
-    cp ../rtl-sdr.rules "$out/etc/udev/rules.d/99-rtl-sdr.rules"
+    install -Dm=644 ../rtl-sdr.rules $out/lib/udev/rules.d/99-rtl-sdr.rules
```

Comment:
According to the documentation both locations `etc/udev/rules.d` and `lib/udev/rules.d` should be valid.

Reply:
Well then, then there's no issue here